lib/buffer.js

```diff
@@ -432,6 +432,16 @@ Object.defineProperty(Buffer.prototype, 'offset', {
   }
 });
 
+const {
+  hexSlice, utf8Slice, asciiSlice, latin1Slice, base64Slice, ucs2Slice
+} = binding;
```

Comment:
Me neither. but what are the alternatives if I want to avoid the cost of property lookups? Splitting this into 6 lines? That’s something I can do if you prefer